museum_of_dreams_project/settings/base.py

```diff
@@ -155,6 +155,7 @@
         "filebrowserBrowseUrl": "/view_bucket_items/",
         "versionCheck": False,
         "language": "en-gb",
+        "staticUrl": f"{STATIC_URL}",
```

Comment:
```suggestion
        "staticUrl": f"/{STATIC_URL}",
```

URL needs to start with a forward slash when running locally, otherwise the browser searches relative to the current page's location.